documentation/docs/services/SSI-bridge/use-cases.md

```diff
@@ -0,0 +1,109 @@
+   can be read using a mobile phone and the Ecommerce-SSI Bridge to verify that the scanner device belongs to a delivery
+   company authorized by the e-commerce operator. This allows verifying authentic deliveries.
+
+### Customer Identity and Delivery Verification
+
+**Problem**: Proof of Collection. Customers and e-commerce sites want to guarantee goods are collected by the right
+customer and avoid threats and frauds in the distribution chain.
+
+1. A customer creates a decentralized identity (DID) using a mobile application. This application can be a standalone
+   credential wallet or an e-commerce shopping app.
+2. The customer performs a purchase on an e-commerce site.
+3. The e-commerce site uses the Ecommerce-SSI Bridge to issue a proof of purchase credential to the customer, which is
+   saved to the customer’s phone.
+4. The customer receives the product delivery and presents the credential in a QR code to the courier scanner.
+5. The courier acquires the credential and uses the Ecommerce-Bridge to verify its authenticity. The delivery is safely
+   handed over to the right customer.
+6. (optional) The customer acquires the courier’s scanner credential (
+   see [ Delivery Company Identity and Scanners Verification]( #delivery-company-identity-and-scanners-verification))
+   and uses the Ecommerce-SSI Bridge to verify that it belongs to an authorized delivery company assuring the customer
+   knows the delivery is legitimate.
+
+_The two scenarios above become even more interesting in the case of automated (i.e., drones) delivery and could even
+include product identification._
+
+## Secure E-commerce Sales
+
+### Customer Identity and Credential Age Verification
+
+**Problem**: Verify a customer’s identity and avoid collecting and storing personal information. This would increase
+compliance and reduce liability for e-commerce and small sellers.
+
+1. An authorized bank employee registers an _organization_ decentralized identity (DID) for its bank.
+2. A customer creates a decentralized identity (DID) using a mobile application which could be a standalone credential
+   wallet or an e-commerce shopping app.
+3. The customer requests an Issuer (e.g., the bank) to issue a credential stating their age.
+4. The issuer uses previously verified information about the user held on local record and the Ecommerce-SSI Bridge to
+   create and issue a Verifiable Credential to the customer.
+5. The customer (namely Owner) downloads the credential in their app using a credential wallet.
+6. The customer purchases an item that requires age verification on an e-commerce site.
+7. The customer provides their credential to the e-commerce website using the Ecommerce-SSI Bridge.
+8. The e-commerce site uses the Ecommerce-SSI Bridge to verify the credential and authorize the purchase.
+
+_A similar scenario can be applied in the online purchase of dedicated drugs for specific health conditions. A GP could
```

Comment:
```suggestion
_A similar scenario can be applied in the online purchase of dedicated drugs for specific health conditions. A general practitioner could
```

Reply:
Just spelling it out for all those Americans who might not know what GP stands for 😂